l10n-pocket/en/footer.ftl

```diff
@@ -35,3 +36,21 @@ pocket-footer-cookie-preferences = Cookie preferences
 pocket-footer-read-it-later-brand-name = { -brand-name-read-it-later-inc }
 pocket-footer-twitter-brand-name = { -brand-name-twitter }
 pocket-footer-facebook-brand-name = { -brand-name-facebook }
+
+### Platform footer is used in some pages:
```

Comment:
Suggestion: change this (and lines below) to a group level comment as opposed to file level
```suggestion
## Platform footer is used in some pages:
```